Client.xcodeproj/xcshareddata/xcschemes/Fennec_Enterprise.xcscheme

```diff
@@ -175,9 +95,9 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Fennec_Enterprise"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
-      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      buildConfiguration = "Fennec"
```

Comment:
We may need Fennec_Enterprise here... we will be signing it for physical device testing... not sure if having Fennec will work...

Reply:
changed back to `Fennec_Enterprise`